install.js

```diff
@@ -231,7 +231,7 @@ installHelpers.getLatestFrameworkVersion(function(error, latestFrameworkTag) {
 function generatePromptOverrides() {
   if(USE_CONFIG) {
     var configJson = require('./conf/config.json');
-    var configData = JSON.parse(JSON.stringify(configJson).replace('true', '"y"').replace('false', '"n"'));
+    var configData = JSON.parse(JSON.stringify(configJson).replace(/true/g, '"y"').replace(/false/g, '"n"'));
```

Comment:
This feels very hacky.

Reply:
This was to deal with how we use y/n for boolean input values (which is the default for prompt). We could maybe look into also accepting true/false as values.

Reply:
I'm happy to look into it now, but as it's an enhancement, I'd prefer to push until 0.5.0 (or even the next patch if we have one)